src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -10,22 +10,22 @@
 public class BattleshipModel {
 
 
```

Comment:
good to see the changes that were needing to be done from the previous assignment